test/integration/install_scripts_int_test.go

```diff
@@ -240,7 +240,7 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstallPerl5_32_Windows() {
 
 	cp := ts.SpawnCmdWithOpts(
 		"powershell.exe",
-		e2e.WithArgs(script, "-t", ts.Dirs.Work, "-activate-default", "ActiveState/Perl-5.32"),
+		e2e.WithArgs(script, "-t", ts.Dirs.Work, "-c", "\"state activate ActiveState/Perl-5.32 --default\""),
```

Comment:
This is still part of important flows used in production for a while, so I'd rather we add another test than replace this one, at least for now.